### PR TITLE
bpo-42392: Update code after merge review from 1st1

### PR DESCRIPTION
**Pull request by uriyyo**
Update code after merge review from @1st1.

Base PR is #23420

https://bugs.python.org/issue42392

### CODE REVIEW

**Comment by uriyyo** [comment]
@1st1 Can you please review this PR and add `skip-news` label?

File: Lib/test/test_asyncio/utils.py [hunk 1]
```diff
@@ -551,8 +551,10 @@ def _get_running_loop():
             frame = sys._getframe(1)
 
             if frame.f_globals['__name__'] == 'asyncio.mixins':
-                # When we called from LoopBoundedMixin we should
-                # fallback to default implementation of get_running_loop
+                # When we called from LoopBoundMixin we should fall back
+                # to the default implementation of get_running_loop because
+                # we must return the currently running event loop
+                # rather than None
```
Comment (1st1) [inline Lib/test/test_asyncio/utils.py]: Ah, now I get it. Perhaps we should refactor our test suite to make sure that `get_running_loop()` works correctly under it, @asvetlov

**Review by 1st1 — APPROVED** [review]

File: Lib/test/test_asyncio/utils.py (hunk 1 above)
Comment (uriyyo) [reply Lib/test/test_asyncio/utils.py]: I thought about it but from what I saw it will require a lot of work so that why I chose an easier way)

Comment (uriyyo) [reply Lib/test/test_asyncio/utils.py]: @1st1 I was curious about what will be if we will remove patching of `_get_running_loop` and when I removed it all tests were passed🤩

Looks like either I missed smth very important, either this patching is no longer required.

Should I update this PR and delete the patching of `_get_running_loop`?

**Comment by uriyyo** [comment]
@aeros Can you please review this PR? You participated in a [discussion](https://github.com/python/cpython/pull/23420/files#r529924924) so I believe your opinion regarding implementation is important.

File: Lib/test/test_asyncio/utils.py (hunk 1 above)
Comment (uriyyo) [reply Lib/test/test_asyncio/utils.py]: @1st1 I have gone ahead and remove `_ger_running_loop`.

Sorry in advance if I violate some rules, I have just thought that if it doesn't break anything then we can remove it.

**Review by aeros — COMMENTED** [review]
Mostly LGTM, the approach looks solid. 

My only suggested improvement would be for `_verify_parameter_is_marker`, since from the current name its purpose in the `__init__` of the primitive classes is not clear at a glance. Here's a couple of possible solutions:

A) Renaming to `_verify_no_loop` and changing it to:

```py
def _verify_no_loop(obj, loop):
    if loop is not _marker:
        raise TypeError(
            f'As of 3.10, the *loop* parameter was removed from '
            f'{type(obj).__name__}() since it is no longer necessary'
        )
```

B) Add a *exc_msg* parameter, so that it's clear what it's being used for. This would look something like:

```py
def _verify_parameter_is_marker(parameter, exc_msg):
    if parameter is not _marker:
        raise TypeError(exc_msg)
```

and then used like:

```py
def __init__(self, ...):
    exc_msg = 'As of 3.10, the *loop* parameter was removed from '
                     f'{type(self).__name__}() since it is no longer necessary'
    _verify_parameter_is_marker(loop, exc_msg)
```

Note that this also removes the necessity of the *obj* parameter since it's only used for getting the function name. 

I'm more in favor of (A) since it seems unlikely to do a similar mass removal of a parameter in the foreseeable future for asyncio and is the most simple, but would be fine w/ (B) since it would still be similarly readable.

File: Lib/test/test_asyncio/utils.py (hunk 1 above)
Comment (1st1) [reply Lib/test/test_asyncio/utils.py]: I think it's better without it.

**Comment by uriyyo** [comment]
@aeros I am more in favor of (A) but (B) is also great because it is a more explicit approach.

Can you please review PR again?
I have updated it using (A) approach that you mentioned.

File: Lib/asyncio/queues.py [hunk 2]
```diff
@@ -17,7 +17,7 @@ class QueueFull(Exception):
     pass
 
 
-class Queue(mixins._LoopBoundedMixin):
+class Queue(mixins._LoopBoundMixin):
```
Comment (asvetlov) [inline Lib/asyncio/queues.py:20]: Why `loop=_marker` trick is not used here?

**Review by asvetlov — CHANGES_REQUESTED** [review]
Looks like you need moving `_verify_no_loop` into `_LoopBoundMixin` as `__init__()` method.

File: Lib/asyncio/queues.py (hunk 2 above)
Comment (uriyyo) [reply Lib/asyncio/queues.py:20]: My bad, fixed

**Comment by uriyyo** [comment]
I have made the requested changes; please review again

**Review by aeros — APPROVED** [review]

**Review by asvetlov — APPROVED** [review]
LGTM

**Pull request merged**